pallets/settlement/src/lib.rs

```diff
@@ -813,7 +820,31 @@ decl_module! {
         #[weight = <T as Trait>::WeightInfo::execute_scheduled_instruction(*legs_count)]
         fn execute_scheduled_instruction(origin, instruction_id: u64, legs_count: u32) {
             ensure_root(origin)?;
-            Self::execute_instruction(instruction_id)?;
+            Self::execute_instruction_retryable(instruction_id)?;
```

Comment:
I noticed `execute_settle_on_affirmation_instruction` does not use the `_retryable` approach. Could you elaborate as to why and ideally note this distinction as a comment in the code as well?